core/domain/state_domain.py

```diff
@@ -797,7 +797,12 @@ class WrittenTranslations(python_utils.OBJECT):
     """
 
     def __init__(self, translations_mapping):
-        """Initializes a WrittenTranslations domain object."""
+        """Initializes a WrittenTranslations domain object.
+
+        Args:
+            translations_mapping: dict. A dict mapping content_id keys to dicts.
+                These dicts map language_code keys to WrittenTranslation objects.
```

Comment:
A couple of comments:
1. Instead of referencing the variable names `content_id` and `language_code`, can you use a description of what those keys represent (e.g. content IDs and language codes)?
2. What is a language code?
3. Right now the antecedent of the "these" in the second sentence is unclear. Maybe combine the sentences with a "which" to make it clear you mean the dicts within the outer dict?